src/GuppyComponents/Utils/queries.js

```diff
@@ -662,8 +662,9 @@ export function getGQLFilter(filterState) {
     filterState === undefined ||
     !('value' in filterState) ||
     Object.keys(filterState.value).length === 0
-  )
-    return undefined;
+  ) {
+    return {};
```

Comment:
This function is used in many other places, I am scared changing the value here could generate bugs somewhere else. could we just update the API call [here](https://github.com/chicagopcdc/data-portal/blob/f6c0e0c179f0a21422cac1e7a965cdb226890f2a/src/GuppyDataExplorer/ExplorerExploreExternalButton/index.jsx#L98) with something like [this](https://github.com/chicagopcdc/data-portal/blob/f6c0e0c179f0a21422cac1e7a965cdb226890f2a/src/redux/explorer/asyncThunks.js#L83)?

Reply:
Yes, that makes sense. Updated, thanks.